includes/Core/Cron.php

```diff
@@ -77,28 +80,35 @@ public function get_remote_data() {
 				'body'    => wp_json_encode( $body ),
 			)
 		);
-
 		$code_response = wp_remote_retrieve_response_code( $response );
 		if ( 200 === $code_response ) {
 			$body_response      = json_decode( wp_remote_retrieve_body( $response ) );
 			$plans              = $body_response->data->plans;
 			$wubtitle_plans     = array();
+			$price_info_plans   = array();
 			$wubtitle_plan_rank = '';
 			$total_jobs         = 0;
 			$total_seconds      = 0;
 			foreach ( $plans as $plan ) {
-				$wubtitle_plans[ $plan->rank ] = array(
+				$wubtitle_plans[ $plan->rank ]   = array(
 					'name'         => $plan->name,
 					'stripe_code'  => $plan->id,
 					// phpcs:disable 
 					// warning camel case
```

Comment:
Instead of putting disable comments on each section, write on top of the file something like this:

```php
// phpcs:disable Rulename
Dove rulename è la regola da ignorare
```